engine/Dockerfile.pg_debug

```diff
@@ -0,0 +1,84 @@
+# syntax = docker/dockerfile:experimental
+
+# Multistage Dockerfile to build the Splitgraph engine: because of all the compilation of
+# FDWs/libraries needed, we have to download a few hundred MB of dev tooling which we don't
+# need at runtime. So, we define a single "toolchain" image, multiple images that build various
+# FDWs and other extensions from it and then the final engine image that cherry-picks required
+# binaries from the previous build stages: this is done by "make installing" them into a different
+# prefix (PGXS's feature) and then copying that over into the filesystem root in the target.
+
+
+#####
+##### toolchain
+#####
+
+FROM postgres:12.3
+
+RUN apt-get update -qq && \
```

Comment:
I feel like most of these apt packages (e.g. libmongoc) aren't needed, can you unify these two apt calls?

Reply:
Sure thing, let me see what's redundant.